Dockerfile

```diff
@@ -1,9 +1,9 @@
-FROM node:12-alpine3.15
+FROM node:16-alpine
 LABEL "repository"="https://github.com/anothrNick/github-tag-action"
```

Comment:
Have you tested this locally?

Reply:
I've built it without issue, but I'm not sure that there is a way to test it properly locally?

It relies on running on Github, for example in the entrypoint there are github-only variables such as $GITHUB_WORKSPACE, $source etc....

relates to #174

Reply:
Let me test this

Reply:
Thanks @sbe-arg , if let me know your local test process, I'll add those instructions to the readme as part of this PR as well!

Reply:
The simplest way is to mock it from a branch.

But I do try to test that semver works as expected in the new node version with 
```
nvm use 16
npm install -g semver
semver -i minor 0.0.0
semver -i major v0.0.0
```

https://github.com/sbe-arg/testing-tags/blob/master/.github/workflows/main.yml#L16

https://github.com/sbe-arg/testing-tags/runs/8297114662?check_suite_focus=true